src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -28,10 +38,30 @@ public void findAllBusinesses() {
     businesses.add(new Listing("The Shiny Bean Coffee & Tea", new MapLocation(40.496328, -79.944862), 4.9, "https://goo.gl/maps/AYH2QCL7pkoMBxHA8", "https://theshinybean.com/"));
     businesses.add(new Listing("Weisshouse", new MapLocation(40.456684, -79.925499), 4.3, "https://goo.gl/maps/7tuXn7QF2hh7ioGYA", "https://www.weisshouse.com/"));
   }
+  
+  // Function is only here until the places API is implemented,
+  // and for test file to be able to set any static business LinkedList we want
+  // to try to use
+  public void setAllBusinesses(List<Listing> allBusinesses){
```

Comment:
Will your test be dependent on this function after the Places API is implemented in findAllBusinesses?

Reply:
I will try to make another test when you tell me @LindseyWeiskopf that the Places API is implemented, this test was just a trial run to see to test the functionality of my function. The only problem is to make a test using an API that needs to connect to the web might be hard as the test always runs before you can local host the server.